andela/travel_tool_front
# #162576739 Remove the command that checks for vulnerabilities

#### What does this PR do?
- Removes the command that checks for vulnerabilities from CircleCI; `npx nsp check`

#### Description of Task to be completed?
Command; 'npx nsp check', that checks for vulnerabilities is randomly failing due to some connectivity issue. Remove it so that CircleCI can smoothly deploy.

#### How should this be manually tested?
Check this link, you would see that it is no longer failing as it was: https://circleci.com/gh/andela/travel_tool_front/11504?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162576739
https://www.pivotaltracker.com/story/show/162576881

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
